### ISSUE TITLE
Keyboard shortcuts don't follow keymap 

### ISSUE THREAD

**Issue by villeneuve** [WayfireWM/wayfire#2000]
**Describe the bug**
On a standard wayfire on a raspberry pi5 running Bookworm
Keyboard is set to FR:     
Section input of file ~/config/wayfire.ini 
```
[input]
xkb_options=
xkb_model=pc105
xkb_layout=fr
xkb_variant=
kb_numlock_default_state = true
```
THIS IS OK IT WORKS. I have FR keyboard in all apps.

The bug is with **shortcuts only**: mismatch between us and fr 
Example this shortcut (hardcoded in wayfire. Not in config files):
\# Close focused window.
close_top_view = \<super\> KEY_Q | \<alt\> KEY_F4
**This shortcut doesn't work with \<super\> KEY_Q but works with \<super\> KEY_A** 
(FR keyboard is AZERTY, US is QWERTY. So Q and A are swapped)
Same problem with this shortcut (defined in ~/config/wayfire.ini ):
binding_netman = \<ctrl\> \<alt\> KEY_W 
command_netman = wfpanelctl netman menu
W and Z swapped

**To Reproduce**
With a french AZERTY keyboard (hardware)
And with keyboard set to FR ( section input ~/config/wayfire.ini as shown here above)
Open any window
Press SUPER + Q 
We expect to close the window but nothing happen
Press SUPER + A
The window closes

**Wayfire version**
0.7.5

Thank you very much. I appreciate wayfire a lot [ former compiz user 10+ years ago ;-)  ]

**Comment by ammen99** [comment]
I think this is a duplicate of #1601 , is it not?

**Comment by villeneuve** [comment]
Yes you are right #1601 is the same issue.
Sorry for that. Before posting I searched with the keyword "keyboard" but missed this one. :-(

Yes on an OS normal user expect to set his keymap once and have it working everywhere.
On Linux for years we had to set the keyboard for the console (the virtual tty) and in Xwindow.

With wayfire on Raspberry Pi OS Bookworm there are 2 files to define your keymap:
~/.config/wayfire.ini
 /usr/share/greeter.ini
Happily the raspi-config tool takes care of both files (after requesting bug correction)
So yes normal user expect to have his keyboard settings working everywhere even with keyboard shortcuts and not to have to deal with several configuration files.

Anyway you said it will be implemented in next release so that's is fine.
Thanks again
